backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1005,8 +1005,8 @@ databaseChangeLog:
                   name: clia_number
                   type: text
                   remarks: The CLIA number for the facility that is ordering (and presumably also conducting) the tests.
-                  constraints:
-                    nullable: false
+                  # constraints:
```

Comment:
Here and the comment below: did something change with our Liquibase? This section is suddenly failing during rollbacks where it hasn't been previously:

```
Caused by: liquibase.exception.DatabaseException: ERROR: column "clia_number" of relation "organization" contains null values [Failed SQL: (0) ALTER TABLE simple_report.organization ADD clia_number TEXT NOT NULL]
```

What it seems to be mad about here is that rows already exist in this table, and you can't impose a `NOT NULL` constraint unless you then give each of those rows a default value - otherwise, those rows would have a null value for the new column and you'd be violating the constraint you just imposed.

What I'm not sure about is what the correct default value would be here.

Reply:
Hmm, I suppose we could use a default value of "0000000000" or similar. On the frontend we have specific CLIA rules that vary by state, but I don't think the database actually has any constraints.

Reply:
hmm we can just remove these instead of commenting them out, 
`clia_number` is currently nullable so I think the rollback was wrong.

Reply:
@zdeveloper just to be clear, you're saying that you don't think we need to back-populate the values here and can leave all of the new column values as `NULL`?